.azure-pipelines.yml

```diff
@@ -145,10 +147,14 @@ stages:
 
   jobs:
   - template: build/stage-release-appstore.yml
+    parameters:    
+      deploymentEnvironment: 'Production'
```

Comment:
We could even have different environment for Store releases

Reply:
I think it's a good idea.